main.cbs.async.js

```diff
@@ -0,0 +1,43 @@
+// ES5 version using Callbacks
+window.versions['cbsasync'] = (function() {
+  'use strict';
+
+  function download(path, cb) {
```

Comment:
remove this, and replace it with `let download = getDownloader('callback');`. See: [download.js#L26-L44](https://github.com/chester1000/js-flows/blob/e96a79c4613355f6d86f0caef09ad6e31cc78fad/download.js#L26-L44)